packages/flutter_tools/lib/src/build_system/targets/common.dart

```diff
@@ -394,3 +395,45 @@ abstract class CopyFlutterAotBundle extends Target {
     environment.buildDir.childFile('app.so').copySync(outputFile.path);
   }
 }
+
+/// Lipo CLI tool wrapper shared by iOS and macOS builds.
+class Lipo {
+  /// Static only.
+  Lipo._();
+
+  /// Create a "fat" binary by combining multiple architecture-specific ones.
+  static Future<void> create(
+      Environment environment,
+      List<DarwinArch> darwinArchs,
+      String path,
+      String inputDir,
+      {bool skipMissingInputs = false}) async {
```

Comment:
Sorry we don't have a linter for this 😞 
```suggestion
  static Future<void> create(
    Environment environment,
    List<DarwinArch> darwinArchs,
    String path,
    String inputDir, {
    bool skipMissingInputs = false,
  }) async {
```